src/main/main/resources/init_db.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE IF NOT EXISTS books;
```

Comment:
would be more stable
```suggestion
DROP SCHEMA IF EXISTS books;
CREATE DATABASE IF NOT EXISTS books;
```